Print a proper error if the version not found in the .ocamlformat file (#390)
ocaml/setup-ocaml: CHANGELOG.md

## [unreleased]

### Fixed

> - Outputs an error if the version is not specified in the `.ocamlformat` file.

Reviewer: ```suggestion
- Print a proper error if the version not found in the `.ocamlformat` file.
```
